### ISSUE TITLE
0.5.12 No Directive annotation found on NgFileSelect

### ISSUE THREAD

**Issue by qq83387856** [jkuri/ng2-uploader#61]
angular rc4

```
import { Component, Input } from "@angular/core";
import { UPLOAD_DIRECTIVES } from "ng2-uploader/ng2-uploader";

@Component({
  selector: 'common-upload',
  template: require('./common-upload.html'),
  directives: [UPLOAD_DIRECTIVES],
})
export class CommonUploadComponent {

  @Input() url: string;

  uploadFile: any;
  options: Object = {
    url: this.url
  };

  handleUpload(data): void {
    if (data && data.response) {
      data = JSON.parse(data.response);
      this.uploadFile = data;
    }
  }
}
```

**Comment by zaneadix** [comment]
Just installed and experiencing the same issue.
To paraphrase: +1

**Comment by doncabron** [comment]
+1 Same problem here

**Comment by mattathompson** [comment]
+1 Same Issue.

**Comment by jkuri** [comment]
Hey guys, I bumped the new version (0.5.13), can you try this please? I made an example and cannot reproduce the problem. Thanks.

**Comment by lacroixdavid1** [comment]
@jkuri Thx a lot, it's all fixed in 0.5.13 ! Working with Angular 2 rc5, webpack2, all with lastest package dependencies

**Comment by jkuri** [comment]
@lacroixdavid1 no problem, good to hear it's working now.
